src/comm.h

```diff
@@ -39,7 +39,6 @@ void comm_import_opened(const Comm::ConnectionPointer &, const char *note, struc
 
 /**
  * Open a port specially bound for listening or sending through a specific port.
- * This is a wrapper providing IPv4/IPv6 failover around comm_openex().
```

Comment:
I removed this comment because it mentioned comm_openex() which became private in this PR. It also read as if comm_open_listener() provides IPv4/IPv6 failover while relying on comm_openex() for the rest of the documented functionality. In reality, it is comm_openex() that provides failover; comm_open_listener() still provides failover indirectly, of course -- no changes there! The failover detail is still mentioned further below in the description blob so we are not losing any critical information here AFAICT.

If others disagree, here is a replacement line that would also work:

```suggestion
 * Provides IPv4/IPv6 failover as detailed below.
```

The above suggestion would need to be applied manually due to GitHub limitations.